### ISSUE TITLE
[Feature/Bug] Calendar loads off of the page

### ISSUE THREAD

**Issue by Human626** [chronotruck/vue-ctk-date-time-picker#188]
First off, this package is great, so thanks. 

I'm not quite sure if this is classified as a bug or a new feature request.

At the moment I'm trying to use your date-time-picker in a modal. Unfortunately when using some laptops (with a smaller screen-size) the date-picker will load beneath the page, then you can't click on the tick/confirm nor can you pick some of the dates (as they aren't visible on the page and you can't scroll down).

You can click away from the modal to select the time/date if it is visible, however a better solution could be to be able to force the calendar always load above the input field.

**Comment by Batsirai** [comment]
Same here

**Comment by Chris-Pratt-Clystnet** [comment]
I dont think this can be solved by the package devs as it isnt really a problem with the calandar as that is loading correctley as it loads at the bottom as it woudnt fit in the space at the top.

What is actually happeing here is that the modal its self is a floating element that follows the page scroll so you can never scroll down to see the rest of the calandar. Also the calandar is a floating element so that dosnt increase the size of the modal to allow the page scroll to work.

Probabley best if you dont use this calandar inside a modal if you want a responsive design.
